### PR TITLE
png and tiff plugins do not depend on C++ exceptions.

### PR DESCRIPTION
**Pull request by philip-lamb**

### CODE REVIEW

File: src/osgPlugins/rot/ReaderWriterROT.cpp [hunk 1]
```diff
@@ -80,7 +80,7 @@ static bool getFilenameAndParams(const std::string& input, std::string& filename
  * This pseudo-loader makes it simple to change the orientation of a saved
  * model by specifying a correcting rotation as part of the filename.
  *
- * Usage: <modelfile.ext>.<rx>,<ry>,<rz>.globe
```
Comment (robertosfield) [inline src/osgPlugins/rot/ReaderWriterROT.cpp]: Why this change and why is part of this PR?

File: src/osgPlugins/trans/ReaderWriterTRANS.cpp [hunk 2]
```diff
@@ -81,7 +81,7 @@ static bool getFilenameAndParams(const std::string& input, std::string& filename
  * This pseudo-loader make it simple to change the origin of a saved model
  * by specifying a correcting translation as part of the filename.
  *
- * Usage: <modelfile.ext>.<tx>,<ty>,<tz>.globe
```
Comment (robertosfield) [inline src/osgPlugins/trans/ReaderWriterTRANS.cpp]: Again, what's going on here?

**Comment by robertosfield** [comment]
Could you explain the specific issue you've seen and how to reproduce it, cutting to the chase to "a solution" doesn't help me work out the most appropriate solution for the problem. Thanks.

File: src/osgPlugins/trans/ReaderWriterTRANS.cpp (hunk 2 above)
Comment (philip-lamb) [reply src/osgPlugins/trans/ReaderWriterTRANS.cpp]: This is correcting what looks like a copy and paste error in the comment describing the usage of the TRANS plugin.

File: src/osgPlugins/rot/ReaderWriterROT.cpp (hunk 1 above)
Comment (philip-lamb) [reply src/osgPlugins/rot/ReaderWriterROT.cpp]: This is correcting what looks like a copy and paste error in the comment describing the usage of the ROT plugin.

**Comment by openscenegraph** [comment]
Thanks for the explanation this helped make sense of the changes.  I've had a look at the png plugin and it does have some throw/catch code but it's safely protected so it's indeed possible to build it without exceptions.  tiff plugin too doesn't have any throw/catch code so will be safe as well.  Comment changes look appropriate as well.
